### PR TITLE
YCSB w/t Couchbase support

### PR DESCRIPTION
**Pull request by maisaengineering**
I was hoping someone from CouchBase would add CouchBase support for YCSB. Apparently missing still.
This fork adds CouchBase support to YCSB. README should help in getting environment up and running.

I am ok if you reject this as this is about 6months old work.

-Chandan Benjaram

### CODE REVIEW

File: couchbase/src/main/java/com/yahoo/ycsb/db/CouchbaseDbClient.java [hunk 1]
```diff
@@ -0,0 +1,347 @@
+/**
```
Comment (busbey) [inline couchbase/src/main/java/com/yahoo/ycsb/db/CouchbaseDbClient.java]: please add license header.

File: couchbase/pom.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (busbey) [inline couchbase/pom.xml]: please add license header

**Comment by busbey** [comment]
#136 has now merged. It looks like this PR has scan support, which we are lacking. If you're up for isolating your change to implementing scan, please let us know.

**Comment by busbey** [comment]
Anyone up for taking on getting the scan support from here integrated into the binding done in #136?

**Comment by kruthar** [comment]
The relevant bits related to scan were merged in #610, this can now be closed.

**Pull request closed without merging**